esphome/components/tuya/light/tuya_light.h

```diff
@@ -44,13 +47,15 @@ class TuyaLight : public Component, public light::LightOutput {
   optional<uint8_t> dimmer_id_{};
   optional<uint8_t> min_value_datapoint_id_{};
   optional<uint8_t> switch_id_{};
+  optional<uint8_t> rgb_id_{};
   optional<uint8_t> color_temperature_id_{};
   uint32_t min_value_ = 0;
   uint32_t max_value_ = 255;
   uint32_t color_temperature_max_value_ = 255;
   float cold_white_temperature_;
   float warm_white_temperature_;
   bool color_temperature_invert_{false};
+  bool color_interlock_{true};
```

Comment:
For consistency with the config schema:
```suggestion
  bool color_interlock_{false};
```